UserShiftsApiService/UserShiftsApiService/Models/SchedulesAndShiftsResponse.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections.Generic;
+
+namespace UserShiftsApiService.Models;
+
+public class SchedulesAndShiftsResponse
+{
+    public List<ScheduleAndShiftsModel> SchedulesAndShifts { get; set; }
```

Comment:
i recommend using native array for any apis because it looks cleaner in some queue applications. ill explain in person